pkg/config/output.go

```diff
@@ -35,14 +35,15 @@ var OutFormats = []string{
 }
 
 type Output struct {
-	Format          string   `mapstructure:"format"`
-	PrintIssuedLine bool     `mapstructure:"print-issued-lines"`
-	PrintLinterName bool     `mapstructure:"print-linter-name"`
-	UniqByLine      bool     `mapstructure:"uniq-by-line"`
-	SortResults     bool     `mapstructure:"sort-results"`
-	SortOrder       []string `mapstructure:"sort-order"`
-	PathPrefix      string   `mapstructure:"path-prefix"`
-	ShowStats       bool     `mapstructure:"show-stats"`
+	Format          string        `mapstructure:"format"` // Deprecated: use Formats instead.
```

Comment:
Should we follow deprecation cycle and add in what version this got deprecated so we remember to remove it? Or are we planning to support this one a bit longer?

Reply:
We have only defined the deprecation cycle for the linters, for the options, I don't know what is the best deprecation cycle.
So we will keep it as long as needed, the implementation is simple to maintain.